src/libs/actions/Report.js

```diff
@@ -892,6 +892,68 @@ function deleteReportComment(reportID, reportAction) {
     API.write('DeleteComment', parameters, {optimisticData, successData, failureData});
 }
 
+/**
+ * @param {String} comment
+ * @returns {Array}
+ */
+const extractLinksInMarkdownComment = (comment) => {
+    const reg = /\[[^[\]]*\]\(([^()]*)\)/gm;
```

Comment:
```suggestion
    const regex = /\[[^[\]]*\]\(([^()]*)\)/gm;
```
While we're fixing variable names, let's make this one `regex`, here and in `removeLinks`